nbviewer/providers/github/handlers.py

```diff
@@ -100,7 +103,8 @@ def get(self, user):
                 url=repo['name'],
                 name=repo['name'],
             ))
-        provider_url = u"https://github.com/{user}".format(user=user)
+  
+        provider_url = u"{github_url}{user}".format(user=user, github_url = _github_url())
```

Comment:
This should likely form the path either with a utility function or as:

```
u"{github_url}/{user}".format(user=user, github_url = _github_url())
```

Normalization ftw.